src/test/java/HomePageTest.java

```diff
@@ -15,6 +23,65 @@ public static void main(String[] arg){
         String title = driver.getTitle();
         Assert.assertEquals(title, "Kwidos");
         driver.quit();
+    }
+
+    @Test
+    public void Homework1() throws InterruptedException {
+        // Go to the homepage https://tstkwidos.tk/
+        // Verify title is equal to kwidos
+        // turn the log off
+        System.setProperty("webdriver.chrome.silentOutput","true");
+        WebDriver driver = new ChromeDriver();
+        driver.get("https://testkwidos.tk/auth/register/contractor");
+        driver.manage().window().maximize();
+        // add the waiting time
+        Thread.sleep(4000);
+        //find the first name filed
+        WebElement firstName = ((ChromeDriver) driver).findElementByCssSelector("input[formcontrolname='firstName']");
```

Comment:
Please use findElement(By.cssSelector()) instead of casting to ChromeDriver and fix this everywhere